compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -2877,7 +2920,7 @@ where
     T: Print<'tcx, P> + TypeFoldable<TyCtxt<'tcx>>,
 {
     fn print(&self, cx: &mut P) -> Result<(), PrintError> {
-        cx.in_binder(self)
+        cx.in_binder(self, WrapBinderMode::ForAll)
```

Comment:
This is always forall, maybe avoid the argument or assert it in the callee so that it doesn't silently get used and not work right or sth